homeassistant/components/hardware/websocket_api.py

```diff
@@ -0,0 +1,38 @@
+"""The Hardware websocket API."""
+from __future__ import annotations
+
+import voluptuous as vol
+
+from homeassistant.components import websocket_api
+from homeassistant.core import HomeAssistant, callback
+
+from .const import DOMAIN
+from .models import HardwareProtocol
+
+
+@callback
+def async_setup(hass: HomeAssistant) -> None:
+    """Set up the hardware websocket API."""
+    websocket_api.async_register_command(hass, ws_info)
+
+
+@websocket_api.websocket_command(
+    {
+        vol.Required("type"): "hardware/info",
+    }
+)
+@callback
+def ws_info(
+    hass: HomeAssistant, connection: websocket_api.ActiveConnection, msg: dict
+) -> None:
+    """Return hardware info."""
+    hardware_info = []
```

Comment:
I think that we can explicit about what the board is. `{"board": board_info}`, there can only be ever one board.

Reply:
I'm happy with whatever floats the boat. The initial commit had a dedicated "board" key, but @zsarnett suggested to just have all hw in a list.

Reply:
In 7e0e97c a dict with a `hardware` list is returned.
The `raspberry_pi` sets the `type` to `board`.